packages/contracts/src/test/plugin/Clonable/PluginCloneableMock.sol

```diff
@@ -14,7 +14,16 @@ contract PluginCloneableV1Mock is PluginCloneable {
     }
 }
 
-contract PluginCloneableV2Mock is PluginCloneable {
+// Doesn't support IPlugin Interface.
+contract PluginCloneableV2Mock {
```

Comment:
I would name the new one `WrongPluginCloneableV1Mock` or something similar. 

The idea behind V1->V2 was to show how a storage variable can be added.
V1->InvalidV2->V3 makes it confusing.

Reply:
https://github.com/aragon/core/pull/242/commits/48b211bbbdf5d20660a929aa53fb37e89ef5c3e9 (resolve if you agree)